src/fetcher.py

```diff
@@ -64,10 +64,14 @@ def __fetch_paginated(self, url: str) -> list[dict]:
 
 def parse_link_header(link_header: str) -> dict[str, str]:
     """Parse the `Link` header from a GitHub API response into a dictionary"""
-    # TODO: implement
-    return {
-        "last": "https://api.github.com/repositories/198065251/commits?per_page=1&page=3095"
-    }
+    chunks = link_header.split(",")
+    output = {}
+    for i in range(0, len(chunks)):
```

Comment:
nit: this would be more readable
```suggestion
    for chunk in chunks:
```